src/modules/deleteTask.js

```diff
@@ -0,0 +1,14 @@
+import listItem from '../listItem.js';
+import todoItems from './task.js';
+
+const deleteTask = function () {
+  todoItems.listItem = this.parentNode;
+  const ul = todoItems.listItem.parentNode;
+  ul.removeChild(todoItems.listItem);
+  localStorage.setItem('deleteTask', JSON.stringify(listItem.todoItems));
+};
+if (JSON.parse(localStorage.getItem('todoItems')) != null) {
+  listItem.todoItems = JSON.parse(localStorage.getItem('deleteTask'));
+}
```

Comment:
- [ ] Kindly in your delete method, you have to remove the task from the tasks array in your local storage, all your tasks will be in an array on the local storage and when you delete a task it should be removed from this array from local storage and from the page also.

![image](https://user-images.githubusercontent.com/35535130/160963247-b2e0f683-5c4e-4d14-b6a1-e4a6af15c62b.png)